docs/root/configuration/http/http_filters/local_rate_limit_filter.rst

```diff
@@ -103,6 +103,88 @@ The route specific configuration:
 Note that if this filter is configured as globally disabled and there are no virtual host or route level
 token buckets, no rate limiting will be applied.
 
+.. _config_http_filters_local_rate_limit_descriptors:
+
+Using Descriptors to rate limit on
+----------------------------------
+
+Descriptors can be used to override local rate limiting based on presence of certain descriptors/route actions.
+A routes :ref:`rate limit action <envoy_v3_api_msg_config.route.v3.RateLimit>` is used to match up a
+:ref:`local descriptor <envoy_v3_api_msg_extensions.common.ratelimit.v3.LocalRateLimitDescriptor>` in the filter config descriptor list.
+The local descriptor's token bucket config is used to decide if request should be
+rate limited or not, if the local descriptor's entries match route's rate limit actions descriptor entries.
+Otherwise default token bucket config is used.
+
+Example filter configuration using descriptors is as follows:
+
+.. code-block:: yaml
```

Comment:
Can you use a `validated-code-block:: yaml` here (see other examples)

Reply:
Fixed